### ISSUE TITLE
Fix cargo fmt

### ISSUE THREAD

**Issue by holygits** [cennznet/cennznet#34]
`cargo fmt` command is failing in this repo with

```rust
thread 'main' panicked at 'internal error: entered unreachable code', src/tools/rustfmt/src/formatting.rs:322:18
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
Exited with code 101
```

The internal error is not our issue but from what I can tell it's caused by the license template feature. Maybe we can consider turning this off for the time being or ignoring the fail code in the PR pipeline on this step for now

**Comment by shaopengw** [comment]
My research update:

The license template bug is fixed in `rustfmt` v1.3.1, but currently the latest version available in `rustup` is v1.3.0. I'm chasing this update issue in rust community.

The currently workaround is to install via source code:
1. Clone `rustfmt` repo, checkout `v1.3.2` tag.
2. `cd` to path of `rustfmt` repo, run `cargo install --path . --force` to install v1.3.2

**Comment by jordy25519** [comment]
Resolved
